spec/index.html

```diff
@@ -3162,6 +3159,54 @@ <h3>Double Circle</h3>
   </section>
 </section>
 
+<section id="canonical-quads" class="appendix">
+  <h2>A Canonical form of N-Quads</h2>
+
+  <p>This section defines a canonical form of N-Quads which has
+    a completely specified layout.
+    The grammar for the language is the same.</p>
```

Comment:
"language is the same": is the same as what? This paragraph seems to come from a different context... Or is there a reference missing?

Reply:
The grammar for the canonical form of N-Quads is the same as the non-canonical grammar for N-Quads.

Reply:
Yep, I thought so:-) But, for my franco-hungarian dialect of English it is not really clear...